apps/meteor/ee/app/livechat-enterprise/server/settings.ts

```diff
@@ -157,6 +157,32 @@ export const createSettings = async (): Promise<void> => {
 		});
 	});
 
+	await settingsRegistry.add('Livechat_AdditionalWidgetScripts', '', {
```

Comment:
What naming convention is being used for these new settings? Shouldn't it be `Livechat_additional_widget_scripts`?